src/js/components/ServicesTable.js

```diff
@@ -252,6 +273,41 @@ var ServicesTable = React.createClass({
     );
   },
 
+  getRestartConfirmDialog: function () {
+    let message = null;
```

Comment:
we don't wanna do this because it'll throw a prop error. `Confirm` requires children. 

see here https://github.com/dcos/dcos-ui/pull/808